.travis.yml

```diff
@@ -21,6 +21,9 @@ before_install:
   - cd coalahtml/_coalahtml
   - bower install
   - cd -
+  # colorama needs to be upgraded to fix error when installed colorama 
+  # version and required are different.
+  - sudo pip3 install -U colorama
```

Comment:
Um maybe we can create an issue to fix this separately? IMHO this needs to be addressed separately?